src/components/header/Header.js

```diff
@@ -1,18 +1,39 @@
 import styled from 'styled-components';
 import {Icon} from '@iconify/react';
 import {bindActionCreators} from 'redux';
-import {useDispatch} from 'react-redux';
+import {useDispatch, useSelector} from 'react-redux';
 import {actionCreators} from '../../state';
+import {useEffect} from 'react';
 
 export default function Header() {
   const {setOpenSideNavigation} = bindActionCreators(actionCreators, useDispatch());
+  const sideNavigationStatus = useSelector(state => state.sideNavigation);
```

Comment:
It's not only status, but it's an object with `status` as a key.
```suggestion
  const sideNavigation = useSelector(state => state.sideNavigation);
```

Reply:
change it, thanks